roles/ceph-facts/tasks/set_radosgw_address.yml

```diff
@@ -1,15 +1,15 @@
 ---
 - name: set_fact _radosgw_address to radosgw_address_block ipv4
   set_fact:
-    _radosgw_address: "{{ hostvars[inventory_hostname]['ansible_all_ipv4_addresses'] | ipaddr(radosgw_address_block) | first }}"
+    _radosgw_address: "{{ hostvars[inventory_hostname]['ansible_all_ipv4_addresses'] | ips_in_ranges(hostvars[item]['monitor_address_block'].split(',')) | first }}"
```

Comment:
didn't catch first the first time but I think it's a wrong copy/paste because you changed the radosgw_address_block to monitor_address_block

Reply:
oh, *facepalm*. Good thing you noticed this!